src/main/java/com/personal/financeManager/documentProcessor/services/impl/DataEntityServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.personal.financeManager.documentProcessor.services.impl;
+
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+import com.couchbase.client.java.json.JsonObject;
+import com.personal.financeManager.documentProcessor.models.DataEntityBean;
+import com.personal.financeManager.documentProcessor.models.SuccessResponse;
+import com.personal.financeManager.documentProcessor.services.DataEntityService;
+
+@Service
+public class DataEntityServiceImpl implements DataEntityService{
+
+	@Override
+	public SuccessResponse createDataEntity(DataEntityBean dataEntityBean) {
+		
+		JSONObject data = new JSONObject(dataEntityBean.getData());
+		
+		String effectiveFromTimestamp = Long.toString(System.currentTimeMillis());
+		String documentType = "2";
+		String version = "000";
+		String documentID = String.format("%s:%s:%s:%s:%s", documentType, dataEntityBean.getFeatureID(),
+				dataEntityBean.getFeatureVariantID(),dataEntityBean.getDataEntityID(), version);
```

Comment:
Please check if DataEntityID is equal to null or empty, if it is empty then just assign the epoch time.